src/app/projects/projects-table/projects-table.component.ts

```diff
@@ -13,16 +14,17 @@ export class ProjectsTableComponent implements OnInit {
   projects: ProjectDto[];
 
   constructor(private router: Router,
-              private projectService: ProjectsService) {
+              private projectsService: ProjectsService,
+              private projectService: ProjectService) {
```

Comment:
trochę słaba nazwa
nazwij to może `StoreService` i się to jakoś połączy z tym, który ja na razie zrobiłem

Reply:
oki